iphone/Classes/TiUIWebView.m

```diff
@@ -387,7 +387,7 @@ - (WKUserScript *)userScriptTitaniumInjectionForAppEvent
     [(TiUIWebViewProxy *)self.proxy setPageToken:_pageToken];
   }
 
-  NSString *source = @"var callbacks = {}; var Ti = {}; Ti.pageToken = %@; \
+  NSString *source = @"var callbacks = {}; var Ti = {}; var Titanium = {}; Ti.pageToken = %@; \
```

Comment:
Wouldn't it be easier to do this instead?
```javascript
var Titanium = Ti;
```

That way you don't have to duplicate the API definitions for the `Titanium` object down below. Both `Ti` and `Titanium` will reference the same object.

Reply:
Yeah. You are right. Updated the PR. Thanks!